src/streams/WritableStream.ts

```diff
@@ -104,7 +112,7 @@ export default class WritableStream<T> {
 	protected _queue: SizeQueue<Record<T>>;
 	protected _writing: boolean;
 
-	constructor(underlyingSink: Sink<T>, strategy: Strategy<T> = {}) {
+	constructor(underlyingSink: Sink<T> = {}, strategy: Strategy<T> = {}) {
```

Comment:
The spec indicates it should default to an empty object:
https://streams.spec.whatwg.org/#ws-class-definition

I know there's not much practical use to it, but without a default value, you can still create a WritableStream, but if you don't provide a sink the constructor throws an unintelligible error (I ran into this while writing tests).

If we are not going to provide a default, then we should at least check the params to see if a sink was provided to the constructor and abort gracefully if not.